apps/publishers/tests/portal/members/test_accept_publisher_member_invite.py

```
@@ -47,3 +47,25 @@ def test_accept_invalid_token_returns_400(self):
         resp = self.client.post("/portal/invitations/not-a-real-token/accept/")
         self.assertEqual(400, resp.status_code, resp.content)
         self.assertEqual("invalid_invitation", resp.json()["error_name"])
+
+    def test_get_invitation_details_no_auth_required(self):
+        _member, inv, raw = self._invite()
```

Ruff (0.15.17)

[warning] Unpacked variable `inv` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)